src/main/java/net/unicon/lti/controller/lti/LTI3Controller.java

```diff
@@ -299,12 +299,12 @@ public String error(HttpServletRequest req) {
             log.debug("Timeout has occurred.");
             log.debug("middlewareState: " + req.getParameter("middlewareState"));
             log.debug("lmsTimedOut: " + req.getParameter("lmsTimedOut"));
-            return "lti3safarierror";
+            return "lti3ErrorMessage";
         } else {
             log.debug("lmsState and middlewareState do not match.");
             log.debug("lmsState: " + req.getParameter("lmsState"));
             log.debug("middlewareState: " + req.getParameter("middlewareState"));
-            return "lti3safarierror";
+            return "lti3ErrorMessage";
```

Comment:
I might be wrong here but I am curious that since we are not replacing all the calls to the existing html file called `lti3error.html`, I am not sure that the new error template `lti3ErrorMessage.html` would currently be used in all the places that would display that original error page. There are a few places where the original error page gets called and you can search the repo for `TextConstants.LTI3ERROR;` to find them. This looks up a constant and resolves to the string `lti3error` which renders that `lti3error.html`  page.

I am wondering if it would it make sense to overwrite the existing html in `lti3error.html` with the work you added into the `lti3ErrorMessage.html` and then we could remove the `lti3ErrorMessage.html`. That way we can keep the BE logic calling the same file in all the places, this would essentially be restyling the whitelabel page with the updated error message. You'd then just need to update the places you switched out the safari error with `return "lti3error"` or `return TextConstants.LTI3ERROR;`

Or another option which would also work, is that we could update the string in `TextConstants.java` line 72 to be `"lti3ErrorMessage" then any place calling `return TextConstants.LTI3ERROR;` would route to the new page.

After reading the AC again I think updating the original `lti3error.html` page seems like a decent route to go but again it might be worth circling back with Sarah if we are unsure.

Reply:
Got it! I wasn't sure if I should replace the content in `lti3error.html` because I wasn't sure where it was being used. But it makes sense to me to update that existing file since it is the whitelabel error. I'm checking with Sarah about the safari error page and I'll make updates accordingly. Thank you!